repos/fdbt-site/src/components/TapPricingRow.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement } from 'react';
+import { ErrorInfo, MultiTapPricing } from '../interfaces';
+import FormElementWrapper, { FormGroupWrapper } from './FormElementWrapper';
+
+interface TapPricingRowProps {
+    numberOfTapsToDisplay: number;
+    errors: ErrorInfo[];
+    userInput: MultiTapPricing[];
+}
+
+export const renderTable = (index: number, errors: ErrorInfo[], userInput: MultiTapPricing[] = []): ReactElement => (
+    <fieldset key={index} className="govuk-fieldset">
+        <legend className="govuk-fieldset__legend govuk-visually-hidden">{`Enter details for tap ${index + 1}`}</legend>
+        <div className="flex-container">
+            <div className="govuk-!-margin-left-4 govuk-!-margin-right-2">
+                <FormGroupWrapper errors={errors} errorIds={[`multi-tap-number-${index}`]}>
```

Comment:
If we remove the form element wrapper below, I think we wont need this group wrapper either?